main.tf

```
@@ -7,6 +7,7 @@ module "default_label" {
 }
 
 module "ecr" {
+  enabled    = "${var.codepipeline_enabled}"
   source     = "git::https://github.com/cloudposse/terraform-aws-ecr.git?ref=tags/0.2.9"
```

```suggestion
  source     = "git::https://github.com/cloudposse/terraform-aws-ecr.git?ref=tags/0.5.0"
```

The previous version did not have `enabled` flag

---

sorry, my bad. I checked master.